specs/ff-asn-selected-filter-cloud/ff-asn-selected-filter-cloud.md

```diff
@@ -1,86 +1,212 @@
+</ff-filter-cloud>
+```
+
+---
+
+### Setup
+```html
+<ff-filter-cloud>
+    <div data-filter-container></div>
+</ff-filter-cloud>
+```
+
+### Rendered HTML
+```html
+<ff-filter-cloud>
+    <div data-filter-container>
+        <span data-filter>Filter 1</span>
+        <span data-filter>Filter 2</span>
+        <!-- ... -->
+        <span data-filter>Filter n</span>
+    </div>
+</ff-filter-cloud>
+```
+
+---
+
+### Setup
+```html
+<ff-filter-cloud>
+    <div data-filter-container>
+        <span data-filter>
+            <span>Custom: {{name}}</span>
+            <span deselect>×</span>
+        </span>
+    </div>
+</ff-filter-cloud>
+```
+
+### Rendered HTML
+```html
+<ff-filter-cloud>
+    <div data-filter-container>
+        <span data-filter>
+            <span>Custom: Filter 1</span>
+            <span deselect>×</span>
+        </span>
+        <span data-filter>
+            <span>Custom: Filter 2</span>
+            <span deselect>×</span>
+        </span>
+        <!-- ... -->
+        <span data-filter>
+            <span>Custom: Filter n</span>
+            <span deselect>×</span>
+        </span>
+    </div>
+</ff-filter-cloud>
+```
+
+
+## Structural attributes
+
+| Attribute             | Required | Description
+| ---------             | -------- | -----------
+| data-filter-container | optional | (TODO) Warning in console if not specified
+| data-filter           | optiional| (TODO) Warning in console if not specified
+| deselect              | optional | (TODO) If not specified, clicking on the element with `data-filter` will trigger the request to deselect the filter.
+
+TODO examples of usage
+
+
 ## API
-### Properties
-| Name | Description |
-| ---- | ----------- |
-| **order**&nbsp;(String) **Options**:&nbsp;"alphabetically,user-selection,fact-finder-group-order",&nbsp; (default: "fact-finder-group-order") | **alphabetically**: Sorts the filter elements alphabetically. <br>**fact-finder-group-order**: Sorts the filter elements in the order provided by FACT-Finder. (Can be configured in the FF UI). <br> **user-selection**: Sorts the filter element in the order the user has clicked them. |
-| **blacklist**&nbsp;(String) | A comma separated list of asn group names which should **not** appear in the list of selected filters. _This attribute can not be used in conjunction with the **whitelist** attribute._ |
-| **whitelist**&nbsp;(String) | A comma separated list of asn group names which should **exclusively** appear in the list of selected filters. _This attribute can not be used in conjunction with the **blacklist** attribute_ |
+
+### Attributes
+
+| Attribute | Required | Type   | Options        | Default     | JS property
+| --------- | -------- | ----   | -------        | -------     | -----------
+| order     | optional | String | _fact-finder_  | fact-finder | `order`
+|           |          |        | alphabetical   |             |
+|           |          |        | user-selection |             |
+
+#### Options
```

Comment:
Could we intend this, to make it more clear, that it refers to the attribute `order`?